r/README.Rmd

```diff
@@ -16,48 +18,77 @@ knitr::opts_chunk$set(
 
 [![conda-forge](https://img.shields.io/conda/vn/conda-forge/r-arrow.svg)](https://anaconda.org/conda-forge/r-arrow)
 
-R integration with Apache Arrow.
+[Apache Arrow](https://arrow.apache.org/) is a cross-language development platform for in-memory data. It specifies a standardized language-independent columnar memory format for flat and hierarchical data, organized for efficient analytic operations on modern hardware. It also provides computational libraries and zero-copy streaming messaging and interprocess communication.
+
+The `arrow` package exposes an interface to the Arrow C++ library.
 
 ## Installation
 
-You first need to install the C++ library:
+Installing `arrow` is a two-step process: first install the Arrow C++ library, then the R package.
 
-### macOS
+### Release version
 
-On macOS, you may install using homebrew:
+The current release of Apache Arrow is 0.13.
 
-```
+On macOS, you may install the C++ library using [Homebrew](https://brew.sh/):
+
+```shell
 brew install apache-arrow
 ```
 
-### From source
+On Linux, see the [Arrow project installation page](http://arrow.apache.org/install/).
+
+Then, you can install the release version of the package from GitHub using the [`remotes`](https://remotes.r-lib.org/) package. From within an R session,
+
+```r
+# install.packages("remotes") # Or install "devtools", which includes remotes
+remotes::install_github("apache/arrow/r", ref="76e1bc5dfb9d08e31eddd5cbcc0b1bab934da2c7")
```

Comment:
Can we use tag name instead of commit ID?

Does `remotes::install_github("apache/arrow/r@apache-arrow-0.13.0")` work?

Reply:
In the future we can, but there were commits after the 0.13.0 arrow release that were needed as Romain, Jeroen, and Javier tried to get the package released and accepted by CRAN, so unfortunately this commit is effectively the 0.13.0 R package release point.

Reply:
Note that any artifact that has not been voted on by the Arrow PMC is not an official release. I think it's fine for CRAN builds to be "unofficial" until we get the development workflow sorted out

Reply:
Noted; just trying to be pragmatic and give installation instructions that work. We'll be sure to keep R on the "official" path going forward.